src/backend/columnar/columnar_tableam.c

```diff
@@ -1193,6 +1429,19 @@ ColumnarSlotCopyHeapTuple(TupleTableSlot *slot)
 }
 
 
+/*
+ * Implementation of TupleTableSlotOps.getsomeattrs for TTSOpsColumnar.
+ */
+static void
+ColumnarSlotGetSomeAttrs(TupleTableSlot *slot, int natts)
+{
```

Comment:
I remember that I got an error and then added a no-op implementation for `getsomeattrs`.
Either I forgot to add a test for that or we don't actually need to have that function, verify which one it is.

Reply:
removing since it looks like we don't need to have that function